code-challenges.test.js

```diff
@@ -1,38 +1,75 @@
-// ASSESSMENT 4: JavaScript Coding Practical Questions with Jest
+// // ASSESSMENT 4: JavaScript Coding Practical Questions with Jest
 
-// Please read all questions thoroughly
-// Pseudo coding is REQUIRED
-// If you get stuck, please leave comments to help us understand your thought process
+// const { describe } = require("yargs");
 
-// Use test driven development to complete the following questions
-// Add appropriate dependencies: $ yarn add jest
+// // Please read all questions thoroughly
+// // Pseudo coding is REQUIRED
+// // If you get stuck, please leave comments to help us understand your thought process
 
-// Reminder: The test will call your function
-// Run the file with the following command: $ yarn jest
+// // Use test driven development to complete the following questions
+// // Add appropriate dependencies: $ yarn add jest
 
-// --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining items.
+// // Reminder: The test will call your function
+// // Run the file with the following command: $ yarn jest
 
-// a) Create a test with an expect statement using the variable provided.
-// HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
+// // --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining items.
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+// // a) Create a test with an expect statement using the variable provided.
+// // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
+// describe ("shuffleRemovingFirst", () => {
+//     it("Create a function that takes in an array, removes the first item from the array and shuffles the remaining items", () => {
+//         const colorsArr1 = shuffleRemovingFirst([...colors1]);
+//         const ColorsArr2 = shuffleRemovingFirst([...colors2]);
+//         expect (colorsArr1).toEqual(expect.arrayContaining(colors1.slice(1)));
```

Comment:
Try to avoid using logic or built-in methods within your tests.  The goal should be testing that when a function is invoked with provided arguments you are getting a set outcome.